ejb-security-programmatic-auth/pom.xml

```diff
@@ -182,7 +199,67 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                  <groupId>org.wildfly.plugins</groupId>
+                  <artifactId>wildfly-maven-plugin</artifactId>
+                  <version>${version.plugin.wildfly}</version>
+              </plugin>
         </plugins>
     </build>
-
+    <profiles>
+        <profile>
+            <id>provisioned-server</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.wildfly.plugins</groupId>
+                        <artifactId>wildfly-maven-plugin</artifactId>
+                        <configuration>
+                            <feature-packs>
+                                <feature-pack>                                    <location>org.wildfly:wildfly-galleon-pack:${version.server}</location>
+                                </feature-pack>
+                            </feature-packs>
+                            <layers>
+                                <!-- layers may be used to customize the server to provision -->
+                                <layer>cloud-server</layer>
+                            </layers>
```

Comment:
think you need to provision the cli script to change the config

Reply:
I added it, but when I run `mvn clean package -Pprovisioned-server`, I get this error: 
```
org.jboss.as.cli.CommandFormatException: Failed to get the list of the operation properties: "WFLYCTL0030: No resource definition is registered for address [
    ("subsystem" => "ejb3"),
    ("application-security-domain" => "quickstart-domain")
]": Failed to get the list of the operation properties: "WFLYCTL0030: No resource definition is registered for address [
    ("subsystem" => "ejb3"),
    ("application-security-domain" => "quickstart-domain")
]"
```
I did not have trouble when I added the configurations for deploying the application.

Reply:
you probably need to add the ejb layer besides the cloud-server

Reply:
I have added it. 
Thanks!